tests/libvmi/BUILD.bazel

```diff
@@ -6,13 +6,16 @@ go_library(
         "cloudinit.go",
         "factory.go",
         "network.go",
+        "status.go",
         "storage.go",
         "vmi.go",
     ],
     importpath = "kubevirt.io/kubevirt/tests/libvmi",
     visibility = ["//visibility:public"],
     deps = [
+        "//pkg/controller:go_default_library",
```

Comment:
Ahhh I do not like this dependency. I guess it is nothing we can do :/

Isn't there some ref between the two without the need to depend on the whole controller?

Reply:
The dependency is introduced with `controller.IsControlledBy` after introduncing `status.go` at `libvmi`, nothing we can quickly do about it.

Reply:
Please refer to the [comment](https://github.com/kubevirt/kubevirt/pull/4493/commits/9f474e5a74eae1860cef00a29e4b2fb38575f22b#r524278427). I think it will solve the issue.